rwengine/src/ai/CharacterController.cpp

```diff
@@ -239,6 +239,28 @@ bool Activities::EnterVehicle::update(CharacterObject *character,
         } else if (character->animator->getAnimation(AnimIndexAction) ==
                    anm_enter) {
             if (character->animator->isCompleted(AnimIndexAction)) {
+                if (vehicle->getVehicle()->vehiclename_ == "TAXI"
+                    || vehicle->getVehicle()->vehiclename_ == "CABBIE"
+                    || vehicle->getVehicle()->vehiclename_ == "BORGNINE") {
+                    // Earn $25 from taxi cab
+                    // @todo
+                } else if (vehicle->getVehicle()->vehiclename_ == "POLICAR") {
+                    // Police cruisers give 5 shotgun cartridges
+                    character->addToInventory(4, 5);
```

Comment:
No idea - I'm sure there'll be some hints somewhere in the codebase.

Please check with the original game wether this might actually be the correct behaviour or prepare to create an issue on merge.